src/components/Outbox.vue

```diff
@@ -0,0 +1,115 @@
+<!--
+  - @copyright Copyright (c) 2022 Richard Steinmetz <richard@steinmetz.cloud>
+  -
+  - @author Richard Steinmetz <richard@steinmetz.cloud>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<AppContent
+		pane-config-key="mail"
+		:show-details="isMessageShown"
+		@update:showDetails="hideMessage">
+		<!-- List -->
+		<template #list>
+			<AppContentList>
+				<Error
+					v-if="error"
+					:error="t('mail', 'Could not open outbox')"
+					message=""
+					role="alert" />
+				<Loading
+					v-else-if="loading"
+					:hint="t('mail', 'Loading messages …')" />
+				<EmptyMailbox v-else-if="messages.length === 0" />
+				<OutboxMessageList
+					v-else
+					:messages="messages" />
```

Comment:
@st3iny i see you removed the messages prop, but now this is not declared and its a required attribute

Reply:
Messages still is a computed property. It was duplicated (prop and computed property) before my changes.